python/ray/scripts/scripts.py

```diff
@@ -508,6 +511,14 @@ def start(node_ip_address, redis_address, address, redis_port, port,
         enable_object_reconstruction=enable_object_reconstruction,
         metrics_export_port=metrics_export_port)
     if head:
+        # Fail early when starting a new cluster when one is already running
+        if address or port is None and not allow_existing:
+            redis_addresses = services.find_redis_address()
+            if len(redis_addresses) > 0:
+                raise ConnectionError(
+                    f"There are existing Ray instances: {redis_addresses}."
+                    " Please specify `--allow-existing` to"
+                    " start multiple clusters.")
```

Comment:
```suggestion
                    " If you intended to start multiple Ray instances on this machine, please specify `--allow-existing` to avoid this error.")
```